build fails on arm64

Building on a Packet c1.large.arm (arm64, ThunderX) system I get this error:

```
error[E0425]: cannot find function `parse_x86_cpuid` in this scope
  --> cranelift-native/src/lib.rs:45:9
   |
45 |         parse_x86_cpuid(&mut isa_builder)?;
   |         ^^^^^^^^^^^^^^^ not found in this scope
error: Could not compile `cranelift-native`.
```

is there arm64 support in this package? I did not expect to get this codepath.

--- a/cranelift-native/src/lib.rs
+++ b/cranelift-native/src/lib.rs
@@ -88,6 +88,11 @@ fn parse_x86_cpuid(isa_builder: &mut isa::Builder) -> Result<(), &'static str> {
     Ok(())
 }
 
+#[cfg(not(any(target_arch = "x86", target_arch = "x86_64")))]
+fn parse_x86_cpuid(_isa_builder: &mut isa::Builder) -> Result<(), &'static str> {
+    unreachable!();
+}
+
 #[cfg(test)]
 mod tests {
     use super::builder;

